public/app/plugins/panel/gauge/module.tsx

```diff
@@ -39,19 +39,33 @@ export const plugin = new PanelPlugin<Options>(GaugePanel)
         description: 'Renders the thresholds as an outer bar',
         defaultValue: defaultOptions.showThresholdMarkers,
       })
+      .addRadio({
+        path: 'sizing',
+        name: 'Gauge size',
+        settings: {
+          options: [
+            { value: BarGaugeSizing.Auto, label: 'Auto' },
+            { value: BarGaugeSizing.Manual, label: 'Manual' },
+          ],
+        },
+        defaultValue: defaultOptions.sizing,
+        showIf: (options: Options) => options.orientation !== VizOrientation.Auto,
+      })
       .addNumberInput({
```

Comment:
Hey @nmarrs Would you mind to replace number input with slider input?
```suggestion
.addSliderInput({
        path: 'minVizWidth',
        name: 'Min width',
        description: 'Minimum column width (vertical orientation)',
        defaultValue: defaultOptions.minVizWidth,
        settings: {
          min: 0,
          max: 300,
          step: 1,
        },
        showIf: (options: Options) =>
          options.sizing === BarGaugeSizing.Manual && options.orientation === VizOrientation.Vertical,
      })
      .addSliderInput({
        path: 'minVizHeight',
        name: 'Min height',
        description: 'Minimum row height (horizontal orientation)',
        defaultValue: defaultOptions.minVizHeight,
        settings: {
          min: 0,
          max: 300,
          step: 1,
        },
        showIf: (options: Options) =>
          options.sizing === BarGaugeSizing.Manual && options.orientation === VizOrientation.Horizontal,
      });
```

Reply:
please, adjust max value as needed 🙂

Reply:
600 max value should be more than enough. In other cases, if e.g. user wants a bigger gauge - auto move should cover that. Let me know what you think.

Reply:
checking it out locally now :) goal is to get this merged today (and associated docs PR) 👍

Reply:
600 seems reasonable enough 👍 updated PR with requested changes